### PR TITLE
use swagger:json command when generating SDK client

### PR DESCRIPTION
**Pull request by jsangilve**
[BC-128]

## What's implemented?

Use a custom command to ouput the swagger.json file required for SDK generation

[BC-128]: https://candis.atlassian.net/browse/BC-128?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by pholuj-stxnext — APPROVED** [review]
Looks fine, but I think we should merge it when other microservices will have implemented "swagger:json" inside package.json.

File: .github/workflows/publish.yaml [hunk 1]
```diff
@@ -52,8 +52,9 @@ jobs:
         if: ${{ env.GENERATE_CLIENT == 'true' }}
         run: |
           npm run build
+          # remove any previously generated version of the SDK
           rm -rf ./packages/sdk/src/fetch-api-client
-          ts-node -r tsconfig-paths/register ./node_modules/@candisio/common-nestjs/lib/cli.js generate-swagger --appModuleSrc=/src/app.module.ts && cd ./packages/sdk && npm run oapi-generate
+          npm run swagger:json && cd ./packages/sdk && npm run oapi-generate
```
[the remark this answers is not in the document]
Comment (jsangilve) [reply .github/workflows/publish.yaml:57]: on it!

[the remark this answers is not in the document]
Comment (jsangilve) [reply .github/workflows/publish.yaml:57]: @grantwilliams sounds like a good idea. I'll take a look into this.

 Given that we don't really build, I guess we can go with conventional commits here as well for the semver?

[the remark this answers is not in the document]
Comment (grantwilliams) [reply .github/workflows/publish.yaml:57]: @jsangilve Yep sounds like that would work well

**Pull request merged**